robottelo/hosts.py

```diff
@@ -499,6 +501,29 @@ def configure_puppet(self, rhel_repo=None, proxy_hostname=None):
         #  this stage can be ignored.
         self.execute('puppet agent -t 2> /dev/null')
 
+    def job_invocation(self, job_template):
+        """Invoke a job on the host by passing name of the job-template.
+
+        :param job-template: name of job-templete to be used.
+        :return: None.
+
+        """
+        invocation_command = make_job_invocation(
+            {'job-template': job_template, 'search-query': f"name ~ {self.hostname}"}
+        )
+        result = JobInvocation.info({'id': invocation_command['id']})
```

Comment:
These classes should be referenced through the `self.cli` namespace.

Reply:
Jobinvocation is not working for me due to unknown reason.
Removing it from this pr and will raise a separate pr once
the issue is fixed. Reverting code to use old way of running openscap scans.